config.js

```diff
@@ -24,5 +26,20 @@ module.exports = {
 		templates: "client/hbs",
 		upload: "upload"
 	},
+	categories:[
```

Comment:
That shouldn't be hard to change, and I kind of agree with that, I think (but it does mean that categories with no posts are auto-hidden, which is kind of nice).

What's the purpose of the name/shortname thing?  Navbar vs. post header?